### PR TITLE
download kubetest manually

### PR DESCRIPTION
**Pull request by haircommander**
and set --get false, in the hopes that getting kubetest ourselves will fix CI woes

Signed-off-by: Peter Hunt <pehunt@redhat.com>

### CODE REVIEW

**Comment by haircommander** [comment]
aaaaaand now we're hitting https://github.com/kubernetes/test-infra/pull/14172 :'(

**Comment by saschagrunert** [comment]
/retest

@haircommander do these failures make sense to me? Does not seem reproducible to me locally...

**Comment by haircommander** [comment]
The before suite failures are a mystery to me...

File: contrib/test/integration/e2e-features.yml [hunk 1]
```diff
@@ -9,7 +9,9 @@
 - name: Buffer the e2e testing command to workaround Ansible YAML folding "feature"
   set_fact:
     e2e_shell_cmd: >
-        KUBE_CONTAINER_RUNTIME="remote" GINKGO_PARALLEL_NODES=6 GINKGO_PARALLEL=y /usr/bin/go run hack/e2e.go
+        KUBE_CONTAINER_RUNTIME="remote" GINKGO_PARALLEL_NODES=6 GINKGO_PARALLEL=y
```
Comment (saschagrunert) [inline contrib/test/integration/e2e-features.yml]: Can we try to run it not in parallel and see if we have a sync problem?

Comment (saschagrunert) [reply contrib/test/integration/e2e-features.yml]: Ah I'm not sure, I think the major issue is that kubedns does not get ready.

File: contrib/cni/10-crio-bridge.conf [hunk 2]
```diff
@@ -5,6 +5,7 @@
     "bridge": "cni0",
     "isGateway": true,
     "ipMasq": true,
+	"hairpinMode": true,
```
Comment (haircommander) [inline contrib/cni/10-crio-bridge.conf]: nit: whitespace

**Comment by haircommander** [comment]
/retest
for kicks and giggles

**Comment by saschagrunert** [comment]
Kubernetes fix for the failing agnhost test: https://github.com/kubernetes/kubernetes/pull/82420

**Comment by haircommander** [comment]
e2e_fedora will be fixed with the above PR
we are deprecating rhel 7 testing, and are actively moving to rhel 8. until we do, we won't pass e2e*rhel. 
Can I get reviews and maybe a force merge (we can wait till fedora passes if we want, or just plow forward. I'm fine with either). regardless, this is an improvement and actually has us run the tests.

**Comment by haircommander** [comment]
@mrunalp @umohnani8 @rhatdan PTAL

**Comment by haircommander** [comment]
Fedora e2e passed, methinks we should merge this

**Pull request merged**